--- /lib/http2/http2.c	2017-10-17 07:21:12.000000000 +0200
+++ /lib/http2/http2.c	2017-10-17 07:21:12.000000000 +0200
@@ -80,6 +80,7 @@
 
 static const char *preface = "PRI * HTTP/2.0\x0d\x0a\x0d\x0aSM\x0d\x0a\x0d\x0a";
 
+/*
 static const char * const h2_state_names[] = {
 	"LWS_H2S_IDLE",
 	"LWS_H2S_RESERVED_LOCAL",
@@ -89,6 +90,7 @@
 	"LWS_H2S_HALF_CLOSED_LOCAL",
 	"LWS_H2S_CLOSED",
 };
+*/
 
 #if 0
 static const char * const h2_setting_names[] = {
